packages/dev/core/src/Misc/observable.ts

```diff
@@ -79,7 +79,7 @@ export class Observer<T> {
      * It will be set by the observable that the observer belongs to.
      * @internal
      */
-    public _remove: Nullable<() => void> = null;
+    public _remove: Nullable<WeakRef<() => void>> = null;

```

Comment:
The problem with the typing will be resolved if this will be changed to a self defined type or "any". This way there is no external reference to WeakRef in our declarations.

Reply:
Oh I see! I will try to do that update if I can find time. Else on Monday

Reply:
resolved in [689264e](https://github.com/BabylonJS/Babylon.js/pull/15863/commits/689264ec957f05588b5520b5068b4c021187badf)